docs/admin/index.rst

```diff
@@ -0,0 +1,154 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   - `Stand alone Gitlab <configurations/standalone-gitlab.html>`_
+   - `Stand alone Keycloak <configurations/standalone-keycloak.html>`_
+
+2. Create and configure Renku PVs and PVCs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All of Renku information is stored in three volumes: Jena, Postgresql and Gitlab (if not stand alone).
+To this end, persistent volumes and persistent volume claims need to be created. You can use the following yaml files as a base to execute the `kubectl` commands below.
+
+   - `Renku PV yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pv.yaml>`_
+   - `Renku PVC yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pvc.yaml>`_
+
+   .. code-block:: console
+
+     $ kubectl create -f renku-pv.yaml
+     $ kubectl create ns renku
+     $ kubectl -n renku create -f renku-pvc.yaml
+     $ kubectl -n renku edit pvc renku-postgresql
+     $ kubectl get pv
+     $ kubectl get pvc -n renku
+     $ kubectl describe persistentvolumeclaim -n renku  ## this should not show any error, just PVs ready to be used
+
+Ensure that PVs have a `default storageclass` and the following value is set:
+
+   .. code-block:: console
+
+     metadata:
+       name: default
+       annotations:
+         storageclass.beta.kubernetes.io/is-default-class: "true"
+
+3. Create a renku-values.yaml file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This step is very important as it will define the values that helm will use to install your Renku instance.
+
+You can find a basic file in `Renku values file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-values.yaml>`_
+
+Ensure the following has been taken care of before deploying Renku:
+
+  - `variables_switchboard` section has URLs and DNS names that correspond to your configuration
+  - `credentials` section has correctly populated values. Please see the comments in the renku-values file for more information.
+  - check the `resources.requests.memory` value makes sense
+  - `lfsObjects` & `registry` require to setup your S3 back-ends, so you need to have that configuration handy and in place
+
+C. Deploying Renku
+------------------
+
+1. (Optional step for standalone Gitlab)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To be able to support notebooks in private projects we need to insert the gitlab sudo token as a secret so that docker can download the corresponding image from the registry.
+
+.. code-block:: bash
+
+  $ kubectl -n renku create secret docker-registry renku-notebooks-registry \\
+  --docker-server=<registryURL>:<port> --docker-username=root \\
+  --docker-password=<gitlabSudoToken> --docker-email=root@renku-mydomain
+
+2. (Optional certificates)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you chose to create a certificate instead of using LetsEncrypt or similar, you can create the TLS secret with the following command:
```

Comment:
```suggestion
If you chose to create a certificate manually instead of using LetsEncrypt or similar, you can create the TLS secret with the following command:
```